src/components/deprecated-warning/deprecated-warning.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+import Link from '../link/link';
+import Tip from '../tip/tip';
+
+export interface DeprecatedWarningProps {
+  alternativeName: string;
+  alternativeLink?: string;
+}
+
+export const DeprecatedWarning: FC<DeprecatedWarningProps> = ({ alternativeName, alternativeLink }) => (
+  <Tip emoji="🚨" title="Deprecated component" type={Tip.types?.DANGER}>
+    <>
+      This is a legacy component and will be deprecated in the next Vibe major version. Please consider using the
```

Comment:
Let's maybe remove `Vibe` from here, so it will be generic and useful for everyone